.gitignore

```diff
@@ -11,3 +11,5 @@ package-lock.json
 /deployer/cloudformation.yml
 CumulusApiDefault.yml
 CumulusApiV1.yml
+aws-config.override.yml
```

Comment:
How do we use an aws-config.override.yml? Would it be possible to put our AWS_ACCOUNT_ID in there?